drone: #namespace
    task_and_robot_environment_name: 'DroneXY-v3'
    ros_ws_abspath: "/home/josu/simulation_ws"
    
    #qlearn parametroak
    alpha: 0.6
    gamma: 0.7
    epsilon: 0.9
    epsilon_discount: 0.998
    nepisodes: 1000
    nsteps: 1100

